django/db/models/fields/computed.py

```diff
@@ -0,0 +1,131 @@
+from django.core import checks
+from django.db import connections
+from django.db.models.sql import Query
+
+from . import Field
+
+__all__ = ["ComputedField"]
+
+
+class ComputedField(Field):
+    computed = True
+    db_returning = True
+
+    _query = None
+    _resolved_expression = None
+    output_field = None
+
+    def __init__(self, expression, output_field=None, persisted=True, *args, **kwargs):
```

Comment:
`persisted` is a slightly odd param name, since it’s in the past tense. We don’t have any other past tense names that I can find in other fields.

I think `persist` would be a better name, or maybe `db_persist`, since it's a database-level thing, like `db_column` ?

Also taking `args` allows duplicate arguments for e.g. `editable`. Let's make this class [keyword-only](https://lukeplant.me.uk/blog/posts/keyword-only-arguments-in-python/):

```suggestion
    def __init__(self, *, expression, output_field=None, persisted=True, **kwargs):
```

(`Field` really should be keyword-only, but it's too late to fix that.)

Reply:
Changed to `db_persist`